.github/workflows/docs.yml

```diff
@@ -10,10 +10,8 @@ name: docs
 
 jobs:
   docs:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
     name: build
-    container:
-      image: ghcr.io/gtk-rs/gtk3-rs/gtk3:latest
```

Comment:
This is required for building, but because of https://github.com/actions/checkout/issues/915 it doesn't work correctly